packages/kbn-custom-integrations/src/components/create/form.tsx

```diff
@@ -187,7 +187,7 @@ export const CreateCustomIntegrationForm = ({
                 <EuiIconTip
                   content={i18n.translate('customIntegrationsPackage.create.dataset.name.tooltip', {
                     defaultMessage:
-                      'Provide a dataset name to help organise these custom logs. This dataset will be associated with the integration. ',
+                      'The name of the dataset associated with this integration. ',
```

Comment:
Do we expect users to understand what a dataset is at this point? If not, it might be helpful to explain why they might want to name it a certain way.